source/release_notes/3.2.12-3.2.rst

```diff
@@ -0,0 +1,64 @@
+.. _3.2.12-3.2:
+
+=====================================
+Percona Server for MongoDB 3.2.12-3.2
+=====================================
+
+Percona is glad to announce the release of
+Percona Server for MongoDB 3.2.12-3.2 on March 8, 2017.
+Downloads are available
+`here <https://www.percona.com/downloads/percona-server-mongodb-3.2>`_
+and from the :ref:`Percona Software Repositories <install>`.
+
+Percona Server for MongoDB is a highly scalable,
+zero-maintenance downtime database
+supporting the MongoDB v3.2 protocol and drivers.
+It extends MongoDB with MongoRocks_,
+:ref:`inmemory`, and :ref:`PerconaFT <perconaft>` storage engines,
+as well as several enterprise features.
+Percona Server for MongoDB requires no changes to MongoDB applications or code.
+
+.. note:: The PerconaFT storage engine has been deprecated
+   and will not be available in future releases.
+
+This release is based on `MongoDB 3.2.12
+<http://docs.mongodb.org/manual/release-notes/3.2/#feb-1-2017>`_
+and includes the following additional fixes:
+
+* :jirabug:`PSMDB-17`: Changed welcome message in the shell
+  to mention |PSMDB| instead of *MongoDB*
+
+* :jirabug:`PSMDB-90`: Added error message for storage engines
+  that do not support :ref:`hot-backup`
+
+* :jirabug:`PSMDB-91`: Deprecated ``audit`` configuration section
+  and added ``auditLog`` instead
+
+* :jirabug:`PSMDB-94`: Fixed ``audit_create_database`` test
```

Comment:
I don't think someone is interested in what tests we fixed, it should not be in the release notes. Please, remove all the tickets mentioning tests.